app/models/post.js

```diff
@@ -18,4 +21,4 @@ const exampleSchema = new mongoose.Schema({
   timestamps: true
 })
 
-module.exports = mongoose.model('Example', exampleSchema)
+module.exports = mongoose.model('Example', postSchema)
```

Comment:
Should this be `'Example'` or `'Post'` or does it matter?

Reply:
Thanks for Catching that, it has been changed 👍